api/src/routes/subscription/tests/request-subscription.test.ts

```diff
@@ -117,7 +119,105 @@ describe('test request subscription route', () => {
 			type: SubscriptionType.Subscriber
 		};
 
-		expect(requestSubscriptionSpy).toHaveBeenCalledWith('testaddress', undefined);
+		expect(requestSubscriptionSpy).toHaveBeenCalledWith('testaddress', seed, presharedKey);
```

Comment:
Just to be sure: you are requesting a subscription with undefined seed and undefined presharedKey, and it still should proceed?

Reply:
yes in this case the seed and presharedKey will be generated by the api :)